File: glp-new-design/count_braces.js
const fs = require('fs');
const content = fs.readFileSync('src/components/AdminDashboard.jsx', 'utf-8');
console.log('CBRACE_OPEN:', content.split('{').length - 1);
console.log('CBRACE_CLOSE:', content.split('}').length - 1);
console.log('PAREN_OPEN:', content.split('(').length - 1);
console.log('PAREN_CLOSE:', content.split(')').length - 1);
console.log('TAG_OPEN:', content.split('<').length - 1);
console.log('TAG_CLOSE:', content.split('>').length - 1);

File: glp-new-design/count_braces.py
with open('src/components/AdminDashboard.jsx', 'r', encoding='utf-8') as f:
    content = f.read()
    print(f'CBRACE_OPEN: {content.count("{")}')
    print(f'CBRACE_CLOSE: {content.count("}")}')
    print(f'PAREN_OPEN: {content.count("(")}')
    print(f'PAREN_CLOSE: {content.count(")")}')
    print(f'TAG_OPEN: {content.count("<")}')
    print(f'TAG_CLOSE: {content.count(">")}')
